integ_test/test_scrapers.py

```diff
@@ -84,7 +84,8 @@ def test_scrape_hungary_hu(put_item, check_common):
     hungary_scraper = HungaryScraper()
     hungary_scraper.scrape()
     item, general, reception = check_common('hungary-hu')
-    assert len(general) == 100
+    # Checking that we're over 30 entries in. It was at 100 before we were removing duplicates and before when we were using pykml to parse data.
```

Comment:
I really meant a comment explaining that this was just a check to make sure we were scraping a minimal amount of data that suggests the website didn't drastically change format (or that a refactoring broke stuff), but okay

Reply:
Ahhh, I see. Changing